apps/store/themes/store/partials/navigation.hbs

```diff
@@ -49,9 +49,9 @@
                             {{/if}}
                         {{/each}}
                     </div>
-                    <div id="tags-collapse" class="tags-collapse" align="center">
+                    <div id="tags-collapse" class="tags-collapse">
                         <p class="small">
-                            <span id="tags-collapse-text">Show all tags&nbsp;&nbsp;</span>
+                            <span id="tags-collapse-text" class="tags-collapse-span">Show all tags&nbsp;&nbsp;</span>
```

Comment:
Try to use CSS rather than `&nbsp;`. Another thing is this will break the CSS functionality.